cdap-proto/src/main/java/co/cask/cdap/proto/NamespaceMeta.java

```diff
@@ -23,22 +23,19 @@
  * Represents metadata for namespaces
  */
 public final class NamespaceMeta {
+  // id is always identical to name. We keep this field here because the UI may depend on it.
```

Comment:
since it seems like you're making updates to UI in this PR, can we not keep this field here then?